### ISSUE TITLE
Changes to Collaborators section

### ISSUE THREAD

**Issue by Preeticp** [fabric8io/fabric8-online-docs#295]
As per the kabal session today, Collaborators section could change to Users and Roles. This should be in post I train.
Some of the changes mentioned:
- 3 types of users: Viewer, Contributor and Admin
- Team level authorization limited to a space
- Security groups with system wide permissions etc.
Note all this is merely in discussion phases as of now, and we can expect a lot of churn before they are implemented.

**Comment by mishaone** [comment]
@Preeticp are you okay to take ownership of this?

**Comment by Preeticp** [comment]
As of now, I am not covering Auth, I thought you were @mishaone. Either ways, I can take the ownership. Eventually this issue, most likely, would need to be split into multiple smaller issues dealing with different services at which point I might need help, depending on my backlog.

**Comment by mishaone** [comment]
Yep, I'm covering whatever is needed, @Preeticp but since you reported this change, I thought you might be the best person to keep an eye on it. I can take it on if you don't have the bandwidth.

**Comment by Preeticp** [comment]
Sounds good, will assign myself. Afaik, it is to be still implemented, so will try to pick up/limit issues keeping this in mind around the time it will be implemented.

**Comment by mishaone** [comment]
Great, thanks! If you find you don't have the bandwidth when there is something actionable for this or any other issue, please do just pass it on or use the help wanted tag.

**Comment by Preeticp** [comment]
Checked with Dev, slated for around middle of J train

**Comment by Preeticp** [comment]
Presentation link: https://docs.google.com/presentation/d/1doJGnM2OlZVWkD9ETq1dJf8Dyf9CoajZRUx_G0x0VJA/edit?usp=sharing

Blue Jeans Link: https://bluejeans.com/s/Os4mi

**Comment by Preeticp** [comment]
WIP doc on Auth related permissions in Planner: https://docs.google.com/spreadsheets/d/1ALVEMyI_i5dvOwx_0ChN9WvlyGhHkCwchylje2SrpdQ/edit#gid=0
As per the meeting, Planner is to be a first client for auth.

Auth scopes in OSIO: https://docs.google.com/spreadsheets/d/1THYCKnc91p2wZKAP6VLZVbqF4bPGOOxuF-q-mrs4gXA/edit#gid=0

**Comment by Preeticp** [comment]
Roles and permissions implemented in Collaboration have been documented via #483 and #482 
However keeping this open as a place holder issue with links to past discussions. Planner side of permissions are likely to be implemented soon.
